### PR TITLE
Update LobbyGui.xaml

### PR DESCRIPTION
**Pull request by Evgencheg**
If possible need to fix loadout to work normally
https://github.com/WWhiteDreamProject/wwdpublic/pull/64

:cl: Spatison
- fix: The size of the Lobby cannot be changed now / Размер лобби теперь нельзя изменить

### CODE REVIEW

**Review by FoxxoTrystan — APPROVED** [review]

**Comment by Evgencheg** [comment]
@FoxxoTrystan do you know how to do the customization menu to be not resizable too?

File: Content.Client/Lobby/UI/LobbyGui.xaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
                  Stretch="KeepAspectCovered" />
     <BoxContainer Name="MainContainer" VerticalExpand="True" HorizontalExpand="True" Orientation="Horizontal"
                   Margin="10 10 10 10" SeparationOverride="2">
-        <SplitContainer State="Auto" HorizontalExpand="True">
```
Comment (DEATHB4DEFEAT) [inline Content.Client/Lobby/UI/LobbyGui.xaml:17]: Can't you use `Resize="False"`?

Comment (DEATHB4DEFEAT) [reply Content.Client/Lobby/UI/LobbyGui.xaml:17]: Oh, I didn't know I could comment on the deleted side, weird

Comment (Evgencheg) [reply Content.Client/Lobby/UI/LobbyGui.xaml:17]: 🤷

Comment (Evgencheg) [reply Content.Client/Lobby/UI/LobbyGui.xaml:17]: okay...

**Comment by Evgencheg** [comment]
@DEATHB4DEFEAT i cant use resizable = false bcs build crashes

**Review by VMSolidus — APPROVED** [review]

**Comment by Evgencheg** [comment]
@VMSolidus can you fix problem with loadouts and noresizable thing?

**Comment by Evgencheg** [comment]
idk how to work with xaml...

**Pull request merged**